apps/admin_app/assets/css/promotions.css

```diff
@@ -0,0 +1,169 @@
+.App {
+    text-align: center;
+  }
+  
+  .App-logo {
+    animation: App-logo-spin infinite 20s linear;
+    height: 40vmin;
+  }
+  
+  .App-header {
+    background-color: #282c34;
+    min-height: 100vh;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    font-size: calc(10px + 2vmin);
+    color: white;
+  }
+  
+  .App-link {
+    color: #61dafb;
+  }
+  
+  .form .formrow{
+    text-align: left;
+    margin:25px auto;
+    max-width: 400px;
+  }
+  
+
+
+  .topnav{
+    background-color: rgba(255,255,255,1);
+    overflow:hidden;
+    margin-top: 20px; 
+    height: 70px;
+  }
+
+  .topnav a{
+    font-size: 30px;
+    float:right;
```

Comment:
@SakinaBoriwala21 this is inconsistent coding again. `font-size: 30px;` is 👍 but `float:right;` not so much. This is again a PR wide issue. please fix this across this PR.

Reply:
Okay